Drop-In/src/test/java/com/braintreepayments/api/VaultManagerActivityUnitTest.java

```diff
@@ -319,6 +319,45 @@ public void onBackPressed_whileLoading_doesNotLeaveActivity() {
         assertFalse(mActivity.isFinishing());
     }
 
+    @Test
+    public void vaultManager_whenVaultedPaymentEdited_returnsToDropIn() {
+//        String authorization = Fixtures.TOKENIZATION_KEY;
```

Comment:
Are these TODO?

Reply:
Also just realizing these should be in `DropInActivityUnitTest`. The name threw me off I kind of blindly copied them here. Will move back!